$OpenBSD: patch-configure,v 1.9 2013/03/30 13:04:46 jasper Exp $

Fix DATADIRNAME: see LOCALBASE/share/aclocal/{glib-gettext.m4,intltool.m4}
Fix some sed (mis)usage.

--- configure.orig	Mon Mar 25 20:57:20 2013
+++ configure	Sat Mar 30 13:33:25 2013
@@ -12456,6 +12456,9 @@ else
 fi
 
     ;;
+    *-*-openbsd*)
+    DATADIRNAME=share
+    ;;
     *)
     DATADIRNAME=lib
     ;;
@@ -15036,7 +15039,7 @@ undef_plugin() {
 		disabled_plugins="$disabled_plugins, $1 ($2)"
 	fi
 
-	PLUGINS=$(echo "$PLUGINS" | sed -e "s/[^a-zA-Z0-9_]$1[ \t\r\n\f]\{1,\}/ /" -e "s/[^a-zA-Z0-9_]$1$//" -e "s/^$1[ \t\r\n\f]\{1,\}//")
+	PLUGINS=$(echo "$PLUGINS" | gsed -e "s/[^a-zA-Z0-9_]$1[ \t\r\n\f]\{1,\}/ /" -e "s/[^a-zA-Z0-9_]$1$//" -e "s/^$1[ \t\r\n\f]\{1,\}//")
 }
 
 # ================================================================
